### PR TITLE
INTYGFV-16119: Added endpoint api/info to get global banners from intygsadmin.

### PR DESCRIPTION
**Pull request by ChrilleSigerhed**
image

### CODE REVIEW

**Review by mikeyOne — APPROVED** [review]
Looks good

File: apps/minaintyg/src/components/Layout/GlobalBanner/GlobalBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { IDSAlertGlobal, IDSIconAttention, IDSIconInformation, IDSIconWarning } from '@frontend/ids-react-ts'
+import { Banner, BannerPriority } from '../../../schema/info.schema'
+
+export function GlobalBanner({ banner }: { banner: Banner }) {
```
Comment (mhornfeldt) [inline apps/minaintyg/src/components/Layout/GlobalBanner/GlobalBanner.tsx:4]: Shouldn't this be a common component between apps and not placed in MI? Maybe there's code from RS that can be reused?

File: apps/minaintyg/src/schema/info.schema.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+export const bannerSchema = z.object({
+  content: z.string(),
+  type: bannerTypeEnum,
+})
+export const infoSchema = z.object({
+  banners: z.array(bannerSchema),
+})
+
+export type Info = z.infer<typeof infoSchema>
```
Comment (mhornfeldt) [inline apps/minaintyg/src/schema/info.schema.ts:19]: Should info be named something less general like InformationResponse or similar?

File: pnpm-lock.yaml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.0'
+lockfileVersion: '6.1'
```
Comment (mhornfeldt) [inline pnpm-lock.yaml:1]: Should this be included in the pr?

File: apps/minaintyg/src/components/Layout/GlobalBanner/GlobalBanner.tsx (hunk 1 above)
Comment (ChrilleSigerhed) [reply apps/minaintyg/src/components/Layout/GlobalBanner/GlobalBanner.tsx:4]: There is a difference in BannerPriority that led me to implement a new Component.

File: apps/minaintyg/src/schema/info.schema.ts (hunk 2 above)
Comment (ChrilleSigerhed) [reply apps/minaintyg/src/schema/info.schema.ts:19]: Good idea!

**Pull request merged**